modules/util/truncate.go

```diff
@@ -14,6 +14,10 @@ const (
 	asciiEllipsis = "..."
 )
 
+func IsLikelySplitLeftPart(s string) bool {
```

Comment:
`IsLikelyTruncated`?

Reply:
Nope, "Truncated" doesn't have such "..."

It matches the original function below "SplitStringAtByteN"

Reply:
I think we can refactor these "truncate" related function in the future together, while keep this PR simple to backport

Reply:
->  Refactor "string truncate" #32984